scheduler/scheduler.h

```diff
@@ -27,4 +27,68 @@
 // Values 0 to 3.
 #define DEBUG_SCHEDULER 0
 
+// The weight the "fastest" scheduler places on using a
+// server with recent statistics over one that has not
+// been used for a while. Values 0 - 128, with higher
+// values meaning the "fastest" servers are used more
+// often.
+#define STATS_UPDATE_WEIGHT 120
```

Comment:
this used to be adjustable from command line in previous commit right? Just curious why it changed?

Reply:
I was starting to get too far into the weeds on how to parse the command-line options when what I wanted to do was offer alternatives to the original algorithm. It's possible such an option will return when it has relevance to more than one algorithm, or when we have an idea of how changing the value might affect the results.